dotCMS/src/integration-test/java/com/dotmarketing/portlets/contentlet/business/ContentletAPITest.java

```diff
@@ -3816,117 +3819,122 @@ public void saveContentWithExistingIdentifier() throws Exception {
+            assertNotNull(ident.getSysExpireDate());
+
+            assertEquals(dateFormat.format(d1), dateFormat.format(ident.getSysPublishDate()));
+            assertEquals(dateFormat.format(d2), dateFormat.format(ident.getSysExpireDate()));
+
+            // if we save another language version for the same identifier
+            // then the identifier should be updated with those dates d3&d4
+            Contentlet c2 = new Contentlet();
+            c2.setStructureInode(testContentType.inode());
+            c2.setStringProperty(textField.variable(), "c2");
+            c2.setIdentifier(c1.getIdentifier());
+            c2.setDateProperty(publishField.variable(), d3);
+            c2.setDateProperty(expireField.variable(), d4);
+            c2.setLanguageId(altlang.getId());
+            c2.setIndexPolicy(IndexPolicy.FORCE);
+            c2 = APILocator.getContentletAPI().checkin(c2, user, false);
+
+            Identifier ident2 = APILocator.getIdentifierAPI().find(c2);
+            assertNotNull(ident2.getSysPublishDate());
+            assertNotNull(ident2.getSysExpireDate());
+
+            assertEquals(dateFormat.format(d3), dateFormat.format(ident2.getSysPublishDate()));
+            assertEquals(dateFormat.format(d4), dateFormat.format(ident2.getSysExpireDate()));
+
+            // the other contentlet should have the same dates if we read it again
+            Contentlet c11 = APILocator.getContentletAPI().find(c1.getInode(), user, false);
+            assertEquals(dateFormat.format(d3),
+                    dateFormat.format(c11.getDateProperty(publishField.variable())));
+            assertEquals(dateFormat.format(d4),
+                    dateFormat.format(c11.getDateProperty(expireField.variable())));
+
+            Contentlet c21 = APILocator.getContentletAPI().find(c2.getInode(), user, false);
+            assertEquals(dateFormat.format(d3),
+                    dateFormat.format(c21.getDateProperty(publishField.variable())));
+            assertEquals(dateFormat.format(d4),
+                    dateFormat.format(c21.getDateProperty(expireField.variable())));
+
+            // also it should be in the index update with the new dates
+            String q = "+structureName:" + testContentType.variable() +
+                    " +inode:" + c11.getInode() +
+                    " +" + testContentType.variable() + "." + publishField.variable() + ":"
+                    + DateUtil
+                    .toLuceneDateTime(d3) +
+                    " +" + testContentType.variable() + "." + expireField.variable() + ":"
+                    + DateUtil
+                    .toLuceneDateTime(d4);
+
+            final long count = APILocator.getContentletAPI().indexCount(q, user, false);
+            assertEquals(1, count);
+        } finally {
+            ContentletTransformer.setUniquePublishExpireDatePerLanguages(uniquePublishExpireDatePerLanguages);
+        }
     }
 
-    @Test
+    ////@Test
```

Comment:
should we delete this test? if not, please explain why commenting it out  @freddyucv